README.md

```diff
@@ -1,5 +1,6 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task-move-spider-DOM/)
+    - [DEMO LINK](https://Vitaliy-Anochin
+.github.io/js_task-move-spider-DOM/)
```

Comment:
please don't for
```suggestion
    - [DEMO LINK](https://Vitaliy-Anochin.github.io/js_task-move-spider-DOM/)
```
this is link ,please try to formatting it right and add it in your PR description